docs_src/microservices/configuration/ConfigurationAndRegistry.md

```diff
@@ -10,14 +10,34 @@
 
 ## Configuration
 
-Please refer to the EdgeX Foundry [architectural decision record](https://github.com/edgexfoundry/edgex-docs/blob/master/docs_src/design/adr/0005-Service-Self-Config.md) for details (and design decisions) behind the configuration in EdgeX.
+Please refer to the following EdgeX Foundry ADRs for details (and design decisions) behind the configuration in EdgeX
+
+- [Sevice Self Config Seeding](../../../design/adr/0005-Service-Self-Config)
+- [Common Configuration](../../../design/adr/0026-Common%20Configuration/)
+
+### Common Configuration
+
+!!! edgey - "EdgeX 3.0"
+    Common configuration in single location is new in Edgex 3.0
+
+Many of each EdgeX service's configuration settings are the same as all other services. 
+These common configuration settings have been consolidating into a single common configuration location which is seeded by the **core-common-config-bootstrapper** service.
```

Comment:
This reads better this way, in my opinion

```suggestion
These common configuration settings have been consolidated into a single common configuration location which is seeded by the **core-common-config-bootstrapper** service.
```

Reply:
fixed